components/esp_modem/include/nmea_parser.h

```diff
@@ -0,0 +1,137 @@
+/*
+ * SPDX-FileCopyrightText: 2015-2022 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Origin: https://github.com/espressif/esp-idf/blob/master/examples/peripherals/uart/nmea0183_parser/main/nmea_parser.h
```

Comment:
Let's just reference the exact commit instead of master, e.g. 
```suggestion
 * Origin: https://github.com/espressif/esp-idf/blob/36f49f36/examples/peripherals/uart/nmea0183_parser/main/nmea_parser.h
```

(agree that it's probably better to simply copy this header here, than reference from IDF/examples)